tests/console/postgresql94server.pm

```diff
@@ -11,21 +11,28 @@
 use base "consoletest";
 use strict;
 use testapi;
+use utils;
 
 sub run() {
     my $self = shift;
 
     select_console 'root-console';
 
     # install the postgresql94 server package
-    assert_script_run "zypper -n in postgresql94-server", 200;
+    zypper_call "in postgresql94-server";
 
     # start the postgresql94 service
-    assert_script_run "/etc/init.d/postgresql start", 200;
+    assert_script_run "systemctl start postgresql.service";
```

Comment:
It worked like this in the 'working example' but it probably makes sense to give it more time :+1: